packages/apps-config/src/api/spec/datahighway-spreehafen-parachain-testnet.ts

```diff
@@ -13,6 +13,7 @@ const definitions: OverrideBundleDefinition = {
       minmax: [0, undefined],
       types: {
         Date: 'i64',
+        AccountInfo: "AccountInfoWithDualRefCount",
```

Comment:
I tried adding and removing this. App works without it due to recent changes. Let me reverify this.